cernopendata/templates/cernopendata_pages/front/main.html

```diff
@@ -0,0 +1,60 @@
+<div class="homepage-section main section align-items-center" data-anchor="main">
+
+    <div class="container container-main ">
+        <div class="main-inner">
+            <div class="cod-banner row">
+                <h2>
+                    <span>Explore more than <span><strong>1 petabyte</strong></span> <br>
+                    <span class="scnd-line">of open particle physics data !</span></span>
+                </h2>
+            </div>
+
+            <div class="cod-search row">
+                <form class="search-form container" action="{{config.THEME_SEARCH_ENDPOINT}}" role="search">
+                    <input type="text" class="form-control search" name="q" placeholder="{{_('Search')}}">
+                    <div class="">
+                        <button type="submit" class="">Search</button>
+                        <div id="cod-home-image">
+                            <img src="{{url_for('static', filename='assets/img/header-image.svg')}}" alt="">
+                        </div>
+                    </div>
+                </form>
+                <div class="search-examples row justify-content-center">
+                    <span class="example-title">search examples:</span>
+                    <a href="{{url_for('invenio_search_ui.search', q = 'collision data') }}">collision datasets</a>,
+                    <a href="{{url_for('invenio_search_ui.search', q = 'configuration') }}">configurations</a>,
+                    <a href="{{url_for('invenio_search_ui.search', q = '7Tev') }}">energy:7TeV</a>
+                </div>
+            </div>
+
+            <div class="cod-collections row justify-content-around">
+                <div class="collections-col justify-content-center">
+                    <header><span>Explore ..</span></header>
+                    <ul>
+                        <li><a href="{{url_for('cernopendata_pages.faceted_search', page='datasets')}}">datasets</a></li>
+                        <li><a href="{{url_for('cernopendata_pages.faceted_search', page='software')}}">software</a></li>
+                        <li><a href="{{url_for('cernopendata_pages.faceted_search', page='vm')}}">environments</a></li>
```

Comment:
- environmens -> search for `...&type=Environment`
- supplementary materials -> search for `...&type=Supplementaries`